### ISSUE TITLE
RSS Feed reader

### ISSUE THREAD

**Issue by dianglesolutions** [OlivierCC/spfx-40-fantastics#97]
hi guys,

did you manage to make it work? For any RSS link i am getting only the spinning 'loading' indicator. Any thoughts?

**Comment by worthy92** [comment]
Same here - getting the same issue

**Comment by joenito5** [comment]
Same here. I was really excited until all i got was the spinning loading wheel

**Comment by Artocol** [comment]
Same here. Any new developments?

**Comment by Cire2302** [comment]
Idem... too many issues with "40 fantastics" webparts...

**Comment by koltyakov** [comment]
@Cire2302 that is what you expect from an unmaintained project published as technology sample, isn't it? Shouldn't be considered as something plug and use without adjusting but a sample and starting point for your own development having an understanding the samples were created 3-2 years back.

I'd suggest taking a look at community samples: [webparts](https://github.com/SharePoint/sp-dev-fx-webparts/tree/master/samples), [extentions](https://github.com/SharePoint/sp-dev-fx-extensions/tree/master/samples) and [some more complex scenarious](https://github.com/SharePoint/sp-dev-solutions/tree/master/solutions) with recent SPFx.

**Comment by msolive1985** [comment]
Update 2020: this particular webpart still not working.

**Comment by murtuzamj** [comment]
> 
> 
> hi guys,
> 
> did you manage to make it work? For any RSS link i am getting only the spinning 'loading' indicator. Any thoughts?

Yes, I was able to get it work! But I had to deploy my own RSS feeds service! the underlying RSS feeds API by Yahoo was deprecated

**Comment by murtuzamj** [comment]
> 
> 
> Update 2020: this particular webpart still not working.

Yes, I was able to get it work! But I had to deploy my own RSS feeds service! the underlying RSS feeds API by Yahoo was deprecated

**Comment by marting1010** [comment]
> > Update 2020: this particular webpart still not working.
> 
> Yes, I was able to get it work! But I had to deploy my own RSS feeds service! the underlying RSS feeds API by Yahoo was deprecated

This would explain why I'm getting the same errors then! @murtuzamj did you develop your own RSS feeds service?

**Comment by murtuzamj** [comment]
Yes, I developed my own RSS feeds REST API (using nodejs), that was taking a JSON containing RSS Urls, and returning Feeds for RSS Urls. 

My Customer wanted to show their blog feeds on their Intranet! Had no other option :-)

All Yahoo services have been deprecated, like YFinance , YFeeds

**Comment by marting1010** [comment]
> Yes, I developed my own RSS feeds REST API (using nodejs), that was taking a JSON containing RSS Urls, and returning Feeds for RSS Urls.
> 
> My Customer wanted to show their blog feeds on their Intranet! Had no other option :-)
> 
> All Yahoo services have been deprecated, like YFinance , YFeeds

Ok thanks @murtuzamj . You've not shared that up here have you? I've found this one as well which I may have a go with [https://github.com/pnp/sp-dev-fx-webparts/tree/main/samples/react-rss-reader](url)

**Comment by murtuzamj** [comment]
No, @marting1010 as I said, it was for a customer.

Check that out please, Does it have a back end proxy to feed the RSS ?
